User Voice settings blank out if no elevenlabs api key

Logs:
```
2025-01-15 15:18:03,679 [MainThread] [ChatDND] [logger] [ERROR] - Exception in Tkinter callback
2025-01-15 15:18:03,679 [MainThread] [ChatDND] [logger] [ERROR] - Traceback (most recent call last):
2025-01-15 15:18:03,679 [MainThread] [ChatDND] [logger] [ERROR] -   File "tkinter\__init__.py", line 1968, in __call__
2025-01-15 15:18:03,679 [MainThread] [ChatDND] [logger] [ERROR] -   File "ui\widgets\member_card.py", line 59, in open_edit_popup
2025-01-15 15:18:03,679 [MainThread] [ChatDND] [logger] [ERROR] -   File "ui\widgets\member_card.py", line 85, in __init__
2025-01-15 15:18:03,679 [MainThread] [ChatDND] [logger] [ERROR] -   File "ui\widgets\member_card.py", line 106, in create_widgets
2025-01-15 15:18:03,679 [MainThread] [ChatDND] [logger] [ERROR] - ValueError: list.remove(x): x not in list
```

Happens on second open
![image](https://github.com/user-attachments/assets/ac0fa875-77eb-4d19-9ed2-49367ac5a0c3)

--- a/src/ui/widgets/member_card.py
+++ b/src/ui/widgets/member_card.py
@@ -102,9 +102,10 @@ class MemberEditCard(ctk.CTkToplevel):
             current_source = db_member.preferred_tts.source
         
         self.tts_source_var = ctk.StringVar(value=current_source)
+        valid_sources = SOURCES[:]
         if not self.config.get(section="ELEVENLABS", option="api_key"):
-            SOURCES.remove(SOURCE_11L)
-        self.tts_source_dropdown = ctk.CTkOptionMenu(self, values=SOURCES, variable=self.tts_source_var, command=self._update_voicelist)
+            valid_sources.remove(SOURCE_11L)
+        self.tts_source_dropdown = ctk.CTkOptionMenu(self, values=valid_sources, variable=self.tts_source_var, command=self._update_voicelist)
 
         voices = self.tts[current_source].get_voices()
         self.tts_options = list(voices.keys())

